apps/src/templates/courseOverview/CourseScript.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component, PropTypes } from 'react';
+import color from "@cdo/apps/util/color";
+import i18n from '@cdo/locale';
+import ProgressButton from '../progress/ProgressButton';
+
+const styles = {
+  main: {
+    width: '100%',
+    height: '100%',
+    background: color.background_gray,
+    borderWidth: 1,
+    borderColor: color.border_gray,
+    borderStyle: 'solid',
+    borderRadius: 2,
+    marginBottom: 12,
+  },
+  content: {
+    padding: 20
+  },
+  description: {
+    marginTop: 20,
+    marginBottom: 20
+  },
+  title: {
+    fontSize: 18,
+    fontFamily: '"Gotham 5r", sans-serif',
+  }
+};
+
+export default class CourseScript extends Component {
```

Comment:
is it preferred to do this all in one line rather than an export default at the bottom of the file?